app/src/organisms/ErrorRecoveryFlows/shared/TwoColLwInfoAndDeck.tsx

```diff
@@ -131,18 +131,16 @@ export function TwoColLwInfoAndDeck(
         const isValidDeck =
           currentLoc != null && newLoc != null && movedLabwareDef != null
 
-        const modulesOnDeck = moduleRenderInfo
-          ?.filter(module => module.targetSlotId != null)
-          .map(module => {
-            return {
-              moduleModel: module.moduleDef.model,
-              moduleLocation: { slotName: module.targetSlotId ?? '' },
-              nestedLabwareDef:
-                module.nestedLabwareId !== failedLwId
-                  ? module.nestedLabwareDef
-                  : null,
-            }
-          })
+        const modulesOnDeck = moduleRenderInfo.map(module => {
```

Comment:
```suggestion
        const modulesOnDeck = moduleRenderInfo?.map(module => {
```
Should fix that failing test! Or you can add a mock so that moduleRenderInfo is not missing when the test runs

Reply:
It seems like the mock is intentionally passing in `null`, to test that we handle "invalid props" correctly? Not sure why, but I'm restoring the `?.` and leaving the mock alone.